### ISSUE TITLE
Occluded windows just went blank

### ISSUE THREAD

**Issue by eamodio** [microsoft/vscode#85338]
Issue Type: <b>Bug</b>

I've only seen this once, but it happened to 2 windows at the same time, while I was editing in a 3rd. There was nothing I could do to make it come back other than restart.

/cc @deepak1556 

VS Code version: Code - Insiders 1.41.0-insider (f24f483428f592a262dc73d68a3f05745c562b31, 2019-11-21T10:43:20.960Z)
OS version: Windows_NT x64 10.0.19028

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz (8 x 4008)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.93GB (17.30GB free)|
|Process Argv|--enable-proposed-api eamodio.remotehub|
|Screen Reader|no|
|VM|0%|
</details>

**Comment by cwadrupldijjit** [comment]
It's happened to all open windows.  It seems like it continues to work and any running debuggers or console processes are still running.  It happened once while I had 6 or 7 windows open at once and all of the windows went blank around the same time.  Also just happened now with only 2 windows open right after I started a process in my terminal.

I only noticed this happening after I updated to the latest insiders version.

**Comment by eamodio** [comment]
It happened again to me, again it was only the background window that it happened to. And when I toggled devtools (via a shortcut key) it fixed the window.

**Comment by cwadrupldijjit** [comment]
Is there a default hotkey for the devtools or do I need to add that myself?

**Comment by eamodio** [comment]
I don't think there is one by default, I've have one setup myself.

**Comment by JacksonKearl** [comment]
I found it happens very consistently when I have a lot of memory pressure, for instance when compiling Code. The window is completely blank, but the pointer will respond to things below (show the select text vs click vs normal glyph), and I can do things like copy text and close the window by clicking where the close button should be.

**Comment by deepak1556** [comment]
I was able to repro this by just opening windows on top of each other and the occluded window gets blank for every new window opened in FIFO order, which aligns with what @eamodio @cwadrupldijjit are seeing, @JacksonKearl scenario seems different but lets verify once I have a fix for this.

**Comment by deepak1556** [comment]
I am also able to repro from out of sources with `release/1.40` branch, definitely a regression from recent electron version bump

**Comment by roblourens** [comment]
Maybe https://github.com/microsoft/vscode/issues/84706 too

**Comment by deepak1556** [comment]
@roblourens that mac issue is slightly different, I know what might cause that , likely https://github.com/electron/electron/issues/21189 but can't confirm a fix for that yet.

**Comment by deepak1556** [comment]
I have narrowed down the cause,

```
diff --git a/src/vs/code/electron-main/window.ts b/src/vs/code/electron-main/window.ts
index b55f4095e3..26b10db860 100644
--- a/src/vs/code/electron-main/window.ts
+++ b/src/vs/code/electron-main/window.ts
@@ -132,7 +132,7 @@ export class CodeWindow extends Disposable implements ICodeWindow {
                                        // want to enforce that Code stays in the foreground. This triggers a disable_hidden_
                                        // flag that Electron provides via patch:
                                        // https://github.com/electron/libchromiumcontent/blob/master/patches/common/chromium/disable_hidden.patch
-                                       backgroundThrottling: false,
+                                       // backgroundThrottling: false,
                                        nodeIntegration: true,
                                        nodeIntegrationInWorker: RUN_TEXTMATE_IN_WORKER,
                                        webviewTag: true
```

reenabling background throttling fixes this.

**Comment by deepak1556** [comment]
Oh btw you don't have to close or toggle devtools for a temporary fix, anything that triggers a repaint of the window will resolve the issue, so simply minimize and restore the window from task bar. A fix is being actively worked on for the next insiders.

**Comment by lediur** [comment]
Running into this issue as well. I started working around it by waiting a few seconds, then unminimizing the window by dragging down and shaking it to get it to repaint properly.

**Comment by chrmarti** [comment]
Fix is in the latest Insiders build (Commit: 599c076d91be1374cf51004cec610f3bcaf4c9cd, Date: 2019-11-22T07:19:06.796Z).

**Comment by wedeluxe** [comment]
As I understand, the issue is supposed to be fixed in VS Code 1.40.2.

But actually, upgrading these two packages
```
code 1.40.1-1 -> 1.40.2-1
electron6 6.1.4-2 -> 6.1.5-1
```
on my Arch Linux PC introduced this issue.
Never saw that in 1.40.1 and before.

My VS Code session has 10 windows open. 6 out of 10 look blank right after starting VS Code. Blank windows still show tooltips when hovering, so content still exists. As suggested above, triggering a repaint of a blank window (restoring, then maximizing again) is a temporary help. And by the way, unblanking one window blanks another.
